### DISCUSSION TITLE
(Fixed) No youtube download again after Aug. 5 update to 9.1.0.42

### DISCUSSION THREAD

**Discussion by BoomerMan54** [aclap-dev/video-downloadhelper#1114] [Help and Q&A] [answered]
### Your question

A week or so ago VDH couldn't download utube videos but worked at other sites.  Auto updates were on, now off, and again can't D/L from utube.  Lic. key verified, FireFox up to date.

I hate having to do a cold boot on my Win10 rig but will if it helps.  Generally just hibernate when shutting down.

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-CA
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"7e091952844b43e1"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "C:\\Video",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 4,
    "successfull_dl": 13,
    "history_limit_in_days": 3
}

**Comment by GrampaWildWilly** [comment]
Give at least one URL of a YouTube video that you can't download.  Don't tell me all.  Give me at least one so I can try it & see if I can replicate your results.

**Reply by BoomerMan54** [reply]
Thanks for responding but I got it working again.  I reverted back to the previous version of VDH and that didn't fix it so had read advice to go into FF settings and clear cache etc so did that then shut down FF and everything else to reboot the PC.  When I started FF again it had lost all my previous tabs with no option in History to restore previous session.  Spent the last couple hours trying to get that back but have given up and just got my pinned tabs restored and have pretty much everything else bookmarked so can find everything again as I need them.

It seems that every time VDH updates I lose the ability to D/L YouTubes until I've done a cold boot of the PC.  I now have auto updates off so I'll do it manually and can avoid FF problems like today.  My fault I think for clearing everything in FF then shutting it down without going to the 3-bar menu or File/exit to shut it down.  I just hit the X at the top right so it didn't save open tabs.

My Win10 PC has run so flawlessly for so long I've got rusty fixing issues when they occur.  Reminds me I haven't done a backup in ages either and my SSD is going on 3 years old.

Thanks again!

**Reply by GrampaWildWilly** [reply]
So now you're running Firefox 129.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20 . . . right?  I've never had to actually boot my entire system after a VDH update.  I've also never had to clear cache & cookies.  Seems like maybe something else strange is going on.  But turning off automatic updates is a good idea.

So is doing backups, but that's a bit off-topic.

**Reply by BoomerMan54** [reply]
I clear out unwanted cookies all the time but first time I cleared all the other stuff which is what I figure lost me my previous tabs listing but I had too many opened I rarely used so maybe a good thing.

I have VDH v9.1.0.12 stable running now so will check for updates just before the next time I plan to shut the PC down .  When I just hibernate for a few weeks things can start getting glitchy which a cold boot sets right again.

I'm just getting back into building rods and tying flies again and have been busy collecting lots of instructional videos to build some sweet custom rods on the 10 blanks I bought from North Fork Composites on one of their sales a few months ago.  Mostly fly rods but next is a 14' 2pc baitcaster with custom cork grips I'm cutting up cork rings to make it with now.  Tedious work but got to have something to do during the long, cold winters up here in northern Alberta in my retirement.  Growing pot has got boring. :)

**Comment by paulrouget** [comment, marked as answer]
Known issue: https://github.com/aclap-dev/video-downloadhelper/discussions/1074
